akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/EventLog.scala

```diff
@@ -26,8 +26,6 @@ private[akka] trait JournalInteractions[C, E, S] {
 
```

Comment:
file name: ExternalInteractions, or two files `JournalInteractions.scala` and `SnapshotInteractions.scala`

Reply:
Updating.

Reply:
I like `ExternalInteractions` better also.